### PR TITLE
First glance through the discord

### PR DESCRIPTION
**Pull request by Pockets-byte**
I took a quick look through the discord and updated the credits with a number of people who were not credited before

### CODE REVIEW

File: docs/credits.html [hunk 1]
```diff
@@ -38,12 +38,29 @@ <h1 style="font-size:50px">The Cortex Command Community Project </h1>
...
+				Contributer - @biiaomongegdr <br>
 				<br>
 				Special People<br>
 				Hero of the Revolution - @Weegee ( n/a )<br>
 				Special Spriters - @High Priestess, @Major<br>
-				Website lead - @.Pockets <br>
-				website style - @???? <br>
+				Website - @Tom <br>
+				old website style - @joshy <br>
+				old website - @. ᛈ ᛟ ᚲ ᚲ ᛖ ᛏ ᛋ <br>
```
Comment (tomflenniken) [inline docs/credits.html]: I don't think the "old" signifier is necessary. I would go with a consistent "Website" credit across the board. Or even move them all over to "Contributor"

Comment (garethyr) [reply docs/credits.html]: I'm thinking a good solution would be to have contributor vs lead in separate sections, so it looks more like
```
Project Leaders
Thing1 - Person1
Thing2 - Person2
Thing3 - Person3
etc.

Project Contributors
C++ Coding - Person1
INI Coding - Person2
Website Design - Person3
etc.
```

**Review by tomflenniken — COMMENTED** [review]
I'm a little concerned about maintenance for a contributors page long-term. For a website like this that doesn't have a bank account behind it, anything that requires a manual change is going to be more likely to not get done.

I don't mean to say that we shouldn't include this change, but we should think about ways to collate existing contributors lists and automate the creation of this page.
The current sources of this data that I know about are:
1. Github contributors across project repos
2. Discord contributor/special tags
3. Google document editors
Though there may be others as well.

File: docs/credits.html (hunk 1 above)
Comment (Pockets-byte) [reply docs/credits.html]: yeah. there's a bit of that already. but it's certainly not done yet. I'm also not sure I've gotten everyone

Comment (garethyr) [reply docs/credits.html]: Can you spend some time changing it and update this then?

**Comment by garethyr** [comment]
Tom, you make a good point, this page will probably end up out of date often. I don't have strong feelings for or against its existence, since everyone will ultimately be credited in the game's credits.

**Comment by Pockets-byte** [comment]
horrendously outdated and very much unneeded

**Pull request closed without merging**